interfaces/apparmor/backend.go

```diff
@@ -145,7 +145,7 @@ func (b *Backend) combineSnippets(snapInfo *snap.Info, opts interfaces.Confineme
 
 func addContent(securityTag string, snapInfo *snap.Info, opts interfaces.ConfinementOptions, snippets map[string][][]byte, content map[string]*osutil.FileState) {
 	policy := defaultTemplate
-	if opts.DevMode && !opts.JailMode {
+	if (opts.DevMode || opts.Classic) && !opts.JailMode {
 		policy = attachPattern.ReplaceAll(policy, attachComplain)
```

Comment:
Alternatively, keep the template but add interfaces/builtin/classic.go with the above rules (and keep ',complain'). classic.go could also use '@unrestricted' for seccomp. The potential downside is this approach makes 'classic' show up in 'snap interfaces'.